Release notes for WAN Assurance 3.7.6 (#638)
128technology/docs: docs/release_notes_wan_assurance_plugin_3.7.md

> ## Release 3.7.5

Reviewer: Need to change the heading here

**Release Date:** November 17, 2023

### New Features
- **WAN-2423 Application Path Insights for conductor managed routers**

Support for the [Application Path Insights](https://www.mist.com/documentation/september-28th-2023-updates/) feature in MIST has been extended to conductor managed routers as part of this release. The feature is enabled on all spoke routers by default upon installing the plugin. The feature can be turned off at the authority and router level if needed.

### Resolved Issues

> - **WAN-2545 Config wiped out after enabling WAN assurance plugin on a router**

Reviewer 2: Suggest: **WAN-2545 Router configuration reset during WAN Assurance onboarding:**

>   _**Resolution:**_ When using MIST redirect onboarding for a router an additional initialization step caused the existing router to be reset causing the configuration to get wiped out. The code paths have been made more smart about the current conductor onboarding status to avoid the un-necessary re-initialization in this scenario.

Reviewer 2: ```suggestion
  _**Resolution:**_ When using MIST redirect onboarding for a router, an additional initialization step caused the existing router to be reset and the configuration removed. Conductor onboarding has been made more robust, avoiding unnecessary reinitialization.
```
